e2e-tests/pitr/conf/restore-on-pitr-minio.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: pxc.percona.com/v1
+kind: PerconaXtraDBClusterRestore
+metadata:
+  name: on-pitr-minio
+spec:
+  pxcCluster: pitr
+  backupName: on-pitr-minio
+#  backupSource:
+#      destination: s3://operator-testing/demand-backup-2020-12-03-09:05:22-full
+#      s3:
+#        bucket: operator-testing
+#        credentialsSecret: gcp-cs-secret
+#        endpointUrl: https://storage.googleapis.com
+#        region: us-east-1
+  pitr:
+    type: latest
+    backupSource:
+      storageName: "minio-binlogs"
```

Comment:
can we use `backupSource` approach for both backup and PITR here?
Reason: to be sure that we can recover from backup to another k8s cluster

Reply:
done https://github.com/percona/percona-xtradb-cluster-operator/pull/697/commits/867e25afaa236becf1253b02c8bf67e3c397219e